endpoints/apiconfig.go

```diff
@@ -765,6 +765,11 @@ func validateRequest(r interface{}) error {
 }
 
 func validateField(v reflect.Value, t reflect.StructField) error {
+	// only validate simple types, ignore arrays, slices, chans, etc.
+	if v.Kind() >= reflect.Array && v.Kind() != reflect.String {
```

Comment:
do we support `Complex64` and `Complex128`. it probably won't make any difference, just checking.